### PR TITLE
Added topic on multiple appliance deployment topologies

### LINKED ISSUE #1356: Document how client handles multiple OVAs on a single VC
Per conversation with @jooskim, we should document how the VIC client handles multiple VIC appliances being registered with the same vCenter Server instance.

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/740, fixes #1356. 

@zjs and @renmaosheng can you please review?

### CODE REVIEW

**Comment by stuclem** [comment]
I added some info about multiple appliances and the plug-in. @wjun can you please review https://github.com/vmware/vic-product/pull/2020/commits/87c43a14e3c98cf9c3536e63b844ae026c8a9764? Thanks!

In combination with info added in https://github.com/vmware/vic-product/pull/2012, https://github.com/vmware/vic-product/pull/2020/commits/87c43a14e3c98cf9c3536e63b844ae026c8a9764 fixes https://github.com/vmware/vic-product/issues/1356.

**Comment by stuclem** [comment]
@reasonerjt would there be any special requirements if you decided to deploy one VIC appliance and then several standalone Harbor instances that you add to projects as additional registries? I'm thinking about user access. Presumably, the standalone Harbor instances would need to be configured to use the same AD setup as the PSC that the appliance is connected to, correct?

File: docs/user_doc/vic_overview/vic_deployment_topos.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+### Two-Way Image Replication <a id="replication"></a>
+
+You can deploy two vSphere Integrated Containers appliances and use the vSphere Integrated Containers Registry instance in each one as the image replication endpoint for the other. 
+
+### User Roles <a id="roles"></a>
+
+If a Platform Services Controller manages multiple vCenter Server instances, you can deploy multiple appliances to different vCenter Server instances that share that Platform Services Controller. This setup provides the following advantages:
+
+- All of the user accounts that are configured in the Platform Services Controller are available to all of the vSphere Integrated Containers appliances. 
+- You can assign diffferent roles to the same user account in different instances of the appliance. For example, a user can be a vSphere Integrated Containers Management Portal administrator for one appliance, but not for another.
```
Comment (reasonerjt) [inline docs/user_doc/vic_overview/vic_deployment_topos.md]: diffferent is a typo

File: docs/user_doc/vic_overview/vic_deployment_topos.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### Hub and Spoke Configuration <a id="hub"></a>
+
+In a large vSphere environment, in which not all vCenter Server instances are located in the same place, or are not all in the same vCenter Single Sign On domain, you can create a hub and spoke configuration: 
+
+- You deploy one vSphere Integrated Containers appliance to a vCenter Server instance in a centralized location. This appliance acts as a hub. 
+- You deploy one vSphere Integrated Containers appliance to each of the  remote vCenter Server instances.
```
Comment (reasonerjt) [inline docs/user_doc/vic_overview/vic_deployment_topos.md]: there one extra space between `the` and `remote`

**Comment by stuclem** [comment]
Thanks @reasonerjt ! Your comments are addressed in https://github.com/vmware/vic-product/pull/2020/commits/4d993167a4ac6b416b5e7d0fd94277043ad025d9.

**Pull request merged**